### DISCUSSION TITLE
Elasticsearch docker

### DISCUSSION THREAD

**Discussion by influence2006** [Security-Onion-Solutions/securityonion#5803] [Unsupported Versions] [answered]
SO 2.3.80 distributed 
I am running into this issue with doing salt highstate. Does anyone know how to resolve this. Seems to be a docker issue? 

```
----------
          ID: so-elasticsearch
    Function: docker_container.running
      Result: False
     Comment: Failed to translate input. Additional info follows:

              invalid:
                  ----------
                  extra_hosts:
                      '8' is not a valid IP address

              If you feel this information is incorrect, the skip_translate argument can be used to skip input translation for the argument(s) identified as invalid. See the documentation for details.
     Started: 14:24:38.067263
    Duration: 32.67 ms
     Changes:
```

**Comment by palichx** [comment, marked as answer]
Hi,
looks like error in salt config. 
Check dockernet in global.sls

**Reply by m0duspwnens** [reply]
@influence2006 This discussion may help resolve the issue: https://github.com/Security-Onion-Solutions/securityonion/discussions/5765#discussioncomment-1436171

**Reply by influence2006** [reply]
Awesome thank you guys! that did work. @m0duspwnens  /opt/so/saltstack/local/pillar/data/nodestab.sls was the solution.
